soh/soh/Enhancements/debugconsole.cpp

```diff
@@ -31,18 +29,22 @@ extern PlayState* gPlayState;
 }
 
 #include <libultraship/bridge.h>
+#include <libultraship/libultraship.h>
 
-#define CMD_REGISTER LUS::GetConsole()->AddCommand
+#define CMD_REGISTER LUS::Context::GetInstance()->GetConsole()->AddCommand
+// TODO: Commands should be using the output passed in.
+#define ERR_MESSAGE std::reinterpret_pointer_cast<LUS::ConsoleWindow>(LUS::Context::GetInstance()->GetWindow()->GetGui()->GetGuiWindow("Console"))->SendErrorMessage
+#define INF_MESSAGE std::reinterpret_pointer_cast<LUS::ConsoleWindow>(LUS::Context::GetInstance()->GetWindow()->GetGui()->GetGuiWindow("Console"))->SendInfoMessage
```

Comment:
Very minor thing, but the abbreviation `INF` has additional meaning in ship, might just bite the bullet and add the extra `O` for clarity

Reply:
good call, might as well just move to `ERROR_MESSAGE` and `INFO_MESSAGE` for readability's sake

Reply:
updated in https://github.com/HarbourMasters/Shipwright/pull/2881/commits/ac2788594f1a2defa14d003160e323e31f4c73cc